x-pack/plugins/logstash/common/constants/editor.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const EDITOR = {
+  /*
+   * The space between the top of the editor's first-line characters
+   * and the top border of the editor.
+   */
+  PIPELINE_EDITOR_SCROLL_MARGIN_TOP: 4,
```

Comment:
I think I missed this in the Grok Debugger PR, but could you add units to the name of these constants, so something like `PIPELINE_EDITOR_SCROLL_MARGIN_TOP_PX` and `PIPELINE_EDITOR_SCROLL_MARGIN_BOTTOM_PX`?

Reply:
Great observation, thanks. I'll add it to all of them.